.github/ISSUE_TEMPLATE/feature_request.md

```diff
@@ -0,0 +1,32 @@
+---
+name: Feature request
+about: Suggest an idea for this project
+title: ''
+labels: feature
+assignees: ''
+
+---
+
+**Is your feature request related to a problem? Please describe.**
+A clear and concise description of what the problem is. Ex. I'm always frustrated when [...]
+
+**Describe the solution you'd like**
+A clear and concise description of what you want to happen, if possible with code examples.
+
+**Describe alternatives you've considered**
+A clear and concise description of any alternative solutions or features you've considered.
+
+**Additional context**
+Add any other context about the feature request here.
+
+**Acceptance Criteria**
+
+- [ ] All checks and tests in the CI pass
+- [ ] Unit tests (90% code coverage or more, [once available](https://github.com/astro-projects/astro/issues/191))
+- [ ] Integration tests (if the feature relates to a new database or external service)
+- [ ] Example DAG 
+- [ ] Docstrings for each of the methods and functions (including Example DAG on how it should be used)
```

Comment:
Should we specify Sphinx syntax here? since going forward we will use it extensively for docs.

Reply:
Absolutely, @utkarsharma2 , do you recommend any particular references on the Sphinx syntax to add a link?

Reply:
I updated the description a bit, @utkarsharma2 , asking for the docstrings to be in `reStructuredText`, since developers could use different standards to write docstrings and autogenerate docs with Sphinx (including the Google Style Python format). Let me know if you'd like to detail it more.